src/components/Thanks/MyKiva/OptInModule.vue

```diff
@@ -82,10 +72,19 @@ const props = defineProps({
 	optedIn: {
 		type: Boolean,
 		default: false
+	},
+	isGuest: {
+		type: Boolean,
+		default: false,
+	},
+	numberOfBadgesEarned: {
```

Comment:
It looks like we aren't passing this property to the component in `ThanksBadges` yet.